bin/forever

```diff
@@ -1,7 +1,19 @@
 #!/usr/bin/env zsh
+# This script runs a command forever. Its frist argument ($refresh_every)
```

Comment:
Typo "frist".

Suggest to add "repeatedly" to the first sentence to clarify that it's multiple invocations.